test/core/create-node.spec.js

```diff
@@ -36,7 +36,7 @@ describe('create node', () => {
         node.stop()
       })
     })
-  })
+  }).timeout(20 * 1000)
```

Comment:
move timeouts up

Reply:
I've seen both ways being used. If we want to stick to timeouts at the top, I think we should switch to using `function` instead of `=>`.